test/python/providers/test_fake_backends.py

```diff
@@ -13,72 +13,18 @@
 # pylint: disable=missing-class-docstring,missing-function-docstring
 # pylint: disable=missing-module-docstring
 
-import datetime
-import itertools
 import operator
-import unittest
-import warnings
-
 from test import combine
 from ddt import ddt, data
 
 from qiskit.circuit import QuantumCircuit
 from qiskit.compiler import transpile
-from qiskit.providers.fake_provider import (
-    Fake5QV1,
-    Fake20QV1,
-    Fake7QPulseV1,
-    Fake27QPulseV1,
-    Fake127QPulseV1,
-    FakeOpenPulse2Q,
-    GenericBackendV2,
-)
-from qiskit.providers.backend_compat import BackendV2Converter, convert_to_target
-from qiskit.providers.models.backendproperties import BackendProperties
-from qiskit.providers.models.backendconfiguration import GateConfig
-from qiskit.providers.backend import BackendV2
+from qiskit.providers.fake_provider import GenericBackendV2
```

Comment:
This test practically became a `GenericBackendV2` test. Would be good to move the test cases here to `test_generic_backend_v2.py`. 
A nitpicking suggestion while you're at it: also move the tests in `tests/python/providers/fake_provider/test_fake_backends.py' to `test_generic_backend_v2.py`.

Reply:
I applied these suggestions and then realized that these are the deprecated interfaces and the tests will be removed in 2.0 anyways, but oh well, at least they look cleaner while they are here :)

Reply:
lol, I could also be more careful about all the other removals and not comment on those files 🔎